t/APISIX.pm

```diff
@@ -493,6 +493,11 @@ _EOC_
 
     $block->set_value("main_config", $main_config);
 
+    # The new directive is introduced here to modify the upper and
```

Comment:
We can modify the schema, not only the upper and lower limits

Reply:
Done